pkg/loghttp/query.go

```diff
@@ -566,12 +566,13 @@ func NewVolumeInstantQueryWithDefaults(matchers string) *logproto.VolumeRequest
 }
 
 type VolumeInstantQuery struct {
-	Start        time.Time
-	End          time.Time
-	Query        string
-	Limit        uint32
-	TargetLabels []string
-	AggregateBy  string
+	Start             time.Time
+	End               time.Time
+	Query             string
+	Limit             uint32
+	TargetLabels      []string
+	AggregateBy       string
+	AggregatedMetrics bool
```

Comment:
nit: I tripped over the naming here. It might be better to use a name like `UseAggregatedMetrics` or `FromAggregatedMetrics`